### ISSUE TITLE
.domain() and Domain suffixes with two parts (i.e. co.uk)

### ISSUE THREAD

**Issue by wasimmer** [medialize/URI.js#17]
Technically, URI.js is most likely returning the correct value, however I was wondering how the following issue could be resolved most elegantly:

``` javascript
URI("http://amazon.co.uk").domain()
"co.uk"
```

... the value I was expecting was "amazon.co.uk". 
So, whenever the suffix of a domain consists out of two parts (co.uk, com.tr) `domain()` returns a value which is not really expected/useful.
Any help for how to treat these composed domain suffixes would be appreciated. Thanks.

**Comment by ul1994** [comment]
Felt I should post this here since I've run into a similar issue.

```
const uri = require('urijs');
const a = new uri('http://torrentz.eu.com');
console.log(a.domain());
```

The above code outputs `eu.com`, but I believe the correct result here should be `torrentz.eu.com`. What would it take to output the expected domain?

**Comment by rodneyrehm** [comment]
@ul1994 I cloned that to #333, fixed the problem and released [v1.18.10](https://github.com/medialize/URI.js/releases/tag/v1.18.10)
